tedana/workflows/tedana.py

```diff
@@ -464,7 +512,20 @@ def tedana_workflow(
     logname = op.join(out_dir, (basename + start_time + "." + extension))
     utils.setup_loggers(logname, repname, quiet=quiet, debug=debug)
 
-    LGR.info("Using output directory: {}".format(out_dir))
+    # Save command into sh file, if the command-line interface was used
+    # TODO: use io_generator to save command
```

Comment:
A "`TODO`" in a PR contents will immediately be flagged by the maintainers. Need to either make the hypothesized modification or remove the comment prior to making the PR to TEDANA.

Reply:
I agree with you. However, this "TODO" is by the Teadan team and as recommeneded by Dan, I only change parts relevant to robustica.

Reply:
This may be due to the same mechanisms as that described in another closed discussion above.

If you look exclusively at the proposed code changes to which the comment is attached, it looks as though the addition of those code comments is a component of this PR and *not* pre-existing content of TEDANA. However this could be due to evolutions in TEDANA and my reorganisation of your code commits happening in parallel. If merging has been done successfully, and the target branch is up to date, then these lines should no longer appear as proposed additions in your PR; and this seems to be the case at time of writing.